ssl test in mysql is flaky

```
> Task :mysql-async:test

com.github.jasync.sql.db.mysql.MySQLSSLConnectionSpec > ssl handler should throws exception when hostname verification fails FAILED
    org.junit.ComparisonFailure at MySQLSSLConnectionSpec.kt:48

116 tests completed, 1 failed, 4 skipped

> Task :mysql-async:test FAILED

FAILURE: Build failed with an exception.
```

--- a/mysql-async/src/test/java/com/github/jasync/sql/db/mysql/ConnectionHelper.kt
+++ b/mysql-async/src/test/java/com/github/jasync/sql/db/mysql/ConnectionHelper.kt
@@ -143,13 +143,10 @@ open class ConnectionHelper : ContainerHelper() {
 
     fun <T> withConfigurableConnection(configuration: Configuration, fn: (MySQLConnection) -> T): T {
         val connection = MySQLConnection(configuration)
-
-        try {
-            awaitFuture(connection.connect())
-            return fn(connection)
-        } finally {
-            awaitFuture(connection.close())
-        }
+        awaitFuture(connection.connect())
+        val res = fn(connection)
+        awaitFuture(connection.close())
+        return res
     }
 
     fun <T> withConfigurableOpenConnection(configuration: Configuration, fn: (MySQLConnection) -> T): T {

